plugins/connectrpc/query-es/v2.0.1/buf.plugin.yaml

```diff
@@ -10,12 +10,12 @@ output_languages:
   - javascript
   - typescript
 registry:
+  opts:
+    - import_extension=js
   npm:
     import_style: module
     rewrite_import_path_suffix: connectquery.js
     deps:
-      - package: '@connectrpc/connect-query'
-        version: ^2.0.1
```

Comment:
Why remove this dep?

Reply:
The generated code doesn't actually depend on it. I think this was a mistake when releasing 2.0.0. Using the output generally means you want connect-query, but this change prevents having multiple versions installed (since you can update connect-query separate from it's plugin).

Reply:
At least, if I understand how deps work in plugins. Please correct me if i'm wrong.

Reply:
Actually, looking at the output of `npm info buf/connectrpc_eliza.connectrpc_query-es`, i expected it to say `@connectrpc/connect-query` and it didn't... so yeah maybe I am confused about what exactly these deps are used for.

Reply:
These deps go into the `package.json`, so when you npm install it pulls in the required deps. 

I haven't followed connect-query to know if the generated code depends on `@connectrpc/connect-query` or not. But if it doesn't, then probably safe to remove. This is where I'd lean on ya'll to know what deps the generated code requires (or not).

Reply:
Ah yeah my confusion was because npm info doesn't list peer deps by default. That makes sense.